_events/2024/2024-08-dei-working-group-call-survey.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Diversity, Equity, and Inclusion Working Group Call
+expires: 2024-08-09
+event_date: "August 8, 2024"
+layout: event
+duration: 60
+repeated: false
+category: dei
+time:
+  - - start: 2024-08-08 14:00 EST
+      end: 2024-08-08 15:00 EST
```

Comment:
EST actually makes the events show up as 1500ET or 1200PT on the add to calendar button, the calendar event, etc. I think you have to use ET instead of EST. Another minor issue with our datetime system

```suggestion
  - - start: 2024-08-08 14:00 ET
      end: 2024-08-08 15:00 ET
```

Reply:
Ooops, my bad on that one. Good catch @exoticDFT. I just removed the TZ altogether and used UTC time.

Updated previews:
* [Event page](https://output.circle-artifacts.com/output/job/8a4f8a90-ffd8-4237-99c4-21b3cedada04/artifacts/0/usrse.github.io/events/2024/2024-08-dei-working-group-call-survey/index.html)
* [List of events](https://output.circle-artifacts.com/output/job/8a4f8a90-ffd8-4237-99c4-21b3cedada04/artifacts/0/usrse.github.io/events/index.html)

The broken URL seems unrelated and I'm not sure of the proper fix atm:
> 🤔 Uh oh... The following urls did not pass:
/github/workspace/_posts/2022-06-04-info-interview-form-rse-group.md:
     ❌️ https://www.nrel.gov/research/staff/rafael-mudafort.html

Reply:
I notified Rafael and he's taking it up with NREL IT.  I propose that it is safe to ignore.